docs/docs/rest-api/public/api/v2/types/pragma.raml

```diff
@@ -39,5 +39,8 @@ annotationTypes:
       Mark a type as internal to indicate, that it is not used in the external API.
       No functional change - only to indicate the use of one type.
       Use only on type level.
-
+  serializeOnly:
+    type: string
```

Comment:
why `string` instead of `nil` type? what's the value of making it a string?

Reply:
better yet, can we come up with a more generic name for what this attribute is actually doing (declaring support for serialization modes)?

for example:
```
serializationModes:
  enum: [read, write, all]
  default: all
```

Reply:
Updated description with an example.

> why `string` instead of `nil` type? what's the value of making it a string?

Just copied the previous pragmas, for sake of consistence.

> better yet, can we come up with a more generic name for what this attribute is actually doing (declaring support for serialization modes)?

Actually, that's not a bad idea :smile: